data-prepper-api/build.gradle

```diff
@@ -5,7 +5,8 @@
 
 dependencies {
     implementation 'io.micrometer:micrometer-core'
-    implementation 'com.fasterxml.jackson.core:jackson-databind:2.13.1'
+    implementation 'com.fasterxml.jackson.core:jackson-databind:2.13.0'
+    implementation "com.fasterxml.jackson.datatype:jackson-datatype-joda"
```

Comment:
Please remove Joda in favor of Java 8 time. Also, when creating strings in Gradle builds, please use single quotes. Using double quotes is actually a GString which is interpolated. It's not wrong, but using single quotes helps with consistency.